ui/selectors/selectors.js

```diff
@@ -828,8 +828,8 @@ function getAllowedAnnouncementIds(state) {
     7: false,
     8: supportsWebHid && currentKeyringIsLedger && currentlyUsingLedgerLive,
     9: false,
-    10: Boolean(process.env.TOKEN_DETECTION_V2) && !process.env.IN_TEST,
-    11: Boolean(process.env.TOKEN_DETECTION_V2) && !process.env.IN_TEST,
+    10: true,
```

Comment:
Why are there two separate notifications for token detection?

Reply:
There is a `Scam and Security risk` announcement and `Improved token Detection` announcement.
<img width="186" alt="Screen Shot 2022-08-09 at 11 06 55 AM" src="https://user-images.githubusercontent.com/43930900/183686726-b2dfeaa5-b673-4198-bad0-f5c95432057c.png">

Reply:
As I was reading through this, I realized that we might want to update the description for the Token detection announcement in extension as we are not keeping token detection ON by default, and token detection with static token still works in MAINNET when token detection toggle is turned OFF.